src/Components/WebAssembly/WebAssembly.Authentication/test/RemoteAuthenticatorCoreTests.cs

```diff
@@ -646,6 +719,13 @@ public TestRemoteAuthenticatorView()
                 };
             }
 
+            public TestRemoteAuthenticatorView(RemoteAuthenticationApplicationPathsOptions applicationPaths,
+                                               IJSRuntime jsRuntime = default)
```

Comment:
```suggestion
            public TestRemoteAuthenticatorView(RemoteAuthenticationApplicationPathsOptions applicationPaths, IJSRuntime jsRuntime = default)
```
nit: We don't indent things this way

Reply:
Thanks!